gravitee-apim-console-webui/src/shared/components/gio-widget-layout/gio-widget-layout.component.ts

```diff
@@ -34,6 +35,7 @@ export class GioWidgetLayoutComponent implements OnChanges {
   tooltip = input<string>();
   state = input.required<GioWidgetLayoutState>();
   errors = input<string[]>();
+  minHeight = input<'small' | 'medium' | 'large'>();
```

Comment:
```suggestion
  minHeight = input<'small' | 'medium' | 'large'>('small');
```
I believe you can add a default here and avoid the `|| 'small'` in the template.